api/go.mod

```diff
@@ -267,7 +268,6 @@ replace (
 
 	github.com/caraml-dev/merlin => github.com/caraml-dev/merlin/api v0.0.0-20240313065547-6778bd14c119
 	github.com/caraml-dev/merlin-pyspark-app => github.com/caraml-dev/merlin/python/batch-predictor v0.0.0-20240313065547-6778bd14c119
-	github.com/caraml-dev/mlp => github.com/deadlycoconuts/mlp v0.0.0-20240917090435-d94d92572eac
```

Comment:
I unfortunately merged this change in my previous PR [here](https://github.com/caraml-dev/turing/pull/394/files#diff-ba0784a950902285760f839a4ad720a381b2981e8a67147f5f43f7508e6343ed), so I'm removing this line in this PR.